### PR TITLE
chore: add count and current page in notification meta details

### LINKED ISSUE #1616: chore: Add count and current page in notification API response
Working on [chatwoot/issues/1489](https://github.com/chatwoot/chatwoot/issues/1489).

API URL —> `<BASE_URL>/api/v1/accounts/:accountId/notifications?page=1`

API Method —> `GET`

- Add the missing details count and current page in the meta object.

### PR DESCRIPTION
**Pull request by muhsin-k**
Fixes #1616

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -456,11 +456,9 @@
     t.index ["taggable_id", "taggable_type", "context"], name: "index_taggings_on_taggable_id_and_taggable_type_and_context"
     t.index ["taggable_id", "taggable_type", "tagger_id", "context"], name: "taggings_idy"
     t.index ["taggable_id"], name: "index_taggings_on_taggable_id"
-    t.index ["taggable_type", "taggable_id"], name: "index_taggings_on_taggable_type_and_taggable_id"
```
Comment (pranavrajs) [inline db/schema.rb:459]: Please revert this changes.

File: db/schema.rb [hunk 2]
```diff
@@ -228,7 +228,7 @@
     t.boolean "locked", default: false
     t.jsonb "additional_attributes", default: {}
     t.bigint "contact_inbox_id"
-    t.uuid "uuid", default: -> { "gen_random_uuid()" }, null: false
```
Comment (pranavrajs) [inline db/schema.rb:231]: Please revert this change.

File: app/controllers/api/v1/accounts/notifications_controller.rb [hunk 3]
```diff
@@ -3,9 +3,11 @@ class Api::V1::Accounts::NotificationsController < Api::V1::Accounts::BaseContro
 
   before_action :fetch_notification, only: [:update]
   before_action :set_primary_actor, only: [:read_all]
+  before_action :set_current_page, only: [:index]
 
   def index
     @unread_count = current_user.notifications.where(account_id: current_account.id, read_at: nil).count
+    @count = current_user.notifications.where(account_id: current_account.id).count
     @notifications = current_user.notifications.where(account_id: current_account.id).page params[:page]
```
Comment (pranavrajs) [inline app/controllers/api/v1/accounts/notifications_controller.rb:11]: Please create a method and memoize `current_user.notifications.where(account_id: current_account.id)`

Comment (pranavrajs) [inline app/controllers/api/v1/accounts/notifications_controller.rb:11]: use @current_page instead of params[:page]

**Review by pranavrajs — CHANGES_REQUESTED** [review]

**Comment by muhsin-k** [comment]
@pranavrajs Please check the latest changes

**Review by pranavrajs — APPROVED** [review]

**Pull request merged**